app/routes/query.$version.tsx

```diff
@@ -2,21 +2,24 @@ import { Link, Outlet, useLocation, useSearchParams } from '@remix-run/react'
 import { json } from '@remix-run/node'
 import type { LoaderFunctionArgs } from '@remix-run/node'
 import { DefaultErrorBoundary } from '~/components/DefaultErrorBoundary'
-import { fetchRepoFile } from '~/utils/documents.server'
 import { repo, getBranch, latestVersion } from '~/projects/query'
 import { RedirectVersionBanner } from '~/components/RedirectVersionBanner'
+import { getTanstackDocsConfig } from '~/utils/config'
 
 export const loader = async (context: LoaderFunctionArgs) => {
   const branch = getBranch(context.params.version)
-  const config = await fetchRepoFile(repo, branch, `docs/config.json`)
+  const tanstackDocsConfig = await getTanstackDocsConfig(repo, branch)
```

Comment:
I noticed that for form and store, they load the config at the docs level (e.g. `/form/latest/docs`), while the other projects load config at the version level (e.g. `/query/latest`. This means that if the config can't be found, these non-docs pages will error. Any chance this could be moved to the docs route? Can definitely be done in a subsequent PR.

Reply:
Oh yeah, no problem! ☺️  These changes are not big, so I'll add them to this PR.